packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js

```diff
@@ -75,3 +74,29 @@ exports.howYouNotifiedPeople = (answers) => {
 	}
 	return notifiedPeople;
 };
+
+/**
+ * @param {{ AppealUser: import('@prisma/client').AppealUser }[]} users
+ * @returns {DataModelUsers}
+ */
+exports.formatUsers = (users) =>
+	users.map(({ AppealUser: { email, isLpaUser } }) => ({
+		salutation: null,
+		firstName: null,
+		lastName: null,
+		emailAddress: email,
+		serviceUserType: isLpaUser ? 'Agent' : 'Appellant'
```

Comment:
We need to send over user detail from the submission.

We'll have one user record and its email

But then we have potentialy two names/orgs 
see db schema: //appellant / agent details

If isAppellant then the contact is appellant, if false its the agent and we also have separate appellant details

We need to change data model to allow org through as well

Reply:
oh so I need to split out contact and appellant from that section in the case it's an agent ye?

Reply:
done now correctly (I hope)